screenshot/demo.py

```diff
@@ -0,0 +1,24 @@
+def insertion_sort(array):
+    breakpoint()
+    i = 0
+    j = 0
+    n = len(array)
+    for j in range(n):
+        key = array[j]
+        i = j - 1
+        while i >= 0 and array[i] > key:
+            array[i + 1] = array[i]
+            i = i - 1
+        array[i + 1] = key
+
+
+if __name__ == "__main__":
+
+    def canonic(self, filename):
+        # fake filename to match original screenshot, as an homage to the original author
+        return '/home/antocuni/env/src/pdb/demo.py'
```

Comment:
Nice idea, but not really necessary I think.

Reply:
thank you for the nice idea but I agree it's not really necessary :)